run_mlp.py

```diff
@@ -0,0 +1,57 @@
+from argparse import ArgumentParser
+
+import numpy as np
+from pytorch_lightning import Trainer
+from torch.utils.data import DataLoader
+
+from dataloader import ChestXrayDataset, ChestXrayTestDataset
+from mlp import LitLinear
+
+BATCH_SIZE = 64
+PATCH_SIZE = 64
+N_PATCHES = 5
+
+
+def run_MLP():
+    parser = ArgumentParser()
+    parser = Trainer.add_argparse_args(parser)
+    args = parser.parse_args(["--gpus=0", "--max_epochs=1", "--max_steps=5", "--val_check_interval=40"])
+    train_dataset = ChestXrayDataset(phase="train", crop=True, patch_size=PATCH_SIZE)
+    val_dataset = ChestXrayDataset(phase="val", crop=True, patch_size=PATCH_SIZE)
+    test_dataset = ChestXrayTestDataset(crop=True, patch_size=PATCH_SIZE, n_per_image=N_PATCHES)
+
+    train_dataloader = DataLoader(train_dataset, batch_size=BATCH_SIZE, shuffle=True)
+    val_dataloader = DataLoader(val_dataset, batch_size=BATCH_SIZE, shuffle=True)
+    test_dataloader = DataLoader(test_dataset, batch_size=1)
```

Comment:
It would be much better (faster) if you can change the later logic so you don't need `batch_size=1` here.